static/docs/user-guide/external-outputs.md

```diff
@@ -19,6 +19,12 @@ an [external cache](/doc/commands-reference/config#cache) location that will be
 used by dvc to store versions of your external file. Non-cached external outputs
 (specified using `-O`) do not require external cache to be setup.
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
+> `dvc fetch` as external cache for your external outputs, because it may cause
+> possible checksum overlaps. `etags` on remote storage might theoretically
```

Comment:
it's not always `etag` it's specific to remote storage. You can say just. `Checksum for some data file on an external storage can potentially collide with checksum generated locally for a different file, with a different content.`

Reply:
updated